languages/rust/exercises/concept/options/tests/options.rs

```diff
@@ -2,85 +2,92 @@ use options::*;
 
 #[test]
 fn test_reviving_dead_player() {
-    let level: u32 = 34;
     let dead_player = Player {
         health: 0,
         mana: Some(0),
-        level,
+        level: 34,
     };
-    let revived_player = dead_player.revive().expect("reviving a dead player must return Some(player)");
+    let revived_player = dead_player
+        .revive()
+        .expect("reviving a dead player must return Some(player)");
     assert_eq!(revived_player.health, 100);
     assert_eq!(revived_player.mana, Some(100));
-    assert_eq!(revived_player.level, level);
+    assert_eq!(revived_player.level, dead_player.level);
 }
 
 #[test]
 #[ignore]
 fn test_reviving_alive_player() {
-    let level: u32 = 8;
     let alive_player = Player {
         health: 1,
         mana: None,
-        level,
+        level: 8,
     };
     assert_eq!(alive_player.revive().is_none(), true);
```

Comment:
Perhaps `assert_eq!(alive_player.revive(), None)` or  `assert!(alive_player.revive().is_none())` ? Probably the former because I think it'd give a more useful result by telling what `Some` was returned, but I guess it doesn't really matter.

Reply:
```suggestion
    assert_eq!(alive_player.revive(), None);
```